### ISSUE TITLE
"Infinite loop" when playing videos

### ISSUE THREAD

**Issue by tszumowski** [WarlaxZ/alexa-home-controller#4]
@WarlaxZ :
I was able to get videos to play using exodus with the "playKodi" intent. However I can't stop the video with the remote by selecting the stop button on the screen. It momentarily stops, but then exodus reloads the sources and starts the video again. 

- This happens no matter how many times I press stop on the remote. 
- This loop continues even if I shut down the alexa-home-controller node.js server.
- It even happens if I load a NEW video using the "playKodi" intent. The server will search for the new movie, but the original movie continues to load/play on Kodi.

Instead, the only way to completely stop the video is to explicitly call the "stop" intent. After that, I have control of my Kodi back.

This is with Kodi Jarvis and tested across a Fire TV as well as a MS Windows installation.

Any ideas?

**Comment by WarlaxZ** [comment]
Yeah this is a bit of a pain, something I am looking to fix, for now the workaround is to just say 'alexa, ask kodi to stop', and it will return to the main menu and prevent the loop. Its basically to do with how I am interacting with exodus, and where I am bypassing the 'pick a source' menu, it unfortunately gets stuck there once it has finished playing. Watch this space though :)

**Comment by agentscript** [comment]
I have been watching this space for 2 months :S 
Any update on this ?
